shared/params/mainnet_config.go

```diff
@@ -139,7 +140,7 @@ var mainnetBeaconConfig = &BeaconChainConfig{
 	GenesisForkVersion:  []byte{0, 0, 0, 0},
 	NextForkVersion:     []byte{0, 0, 0, 0}, // Set to GenesisForkVersion unless there is a scheduled fork
 	NextForkEpoch:       1<<64 - 1,          // Set to FarFutureEpoch unless there is a scheduled fork.
-	ForkVersionSchedule: map[uint64][]byte{
-		// Any further forks must be specified here by their epoch number.
+	ForkVersionSchedule: map[uint64]*pb.Fork{
```

Comment:
This would be a set of fully populated forks. Example:
```go
ForkVersionSchedule: map[uint64]*pb.Fork{
  5000: &pb.Fork{ // At state transition for slot 5000, insert this fork.
    PreviousVersion: 0x0000, // genesisVersion
    CurrentVersion: 0x0001,
    Epoch: 200, // Fork takes effect in epoch 200
  },
  10000: &pb.Fork{
    PreviousVersion: 0x0001, // Previous fork version as defined in this list.
    CurrentVersion: 0x0002,
    Epoch: 325,
  },
}
```